### DISCUSSION TITLE
Hexo-侧边栏 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#332] [Announcements]
# Hexo-侧边栏 | Zi-Zi's Journey

在正文旁边加点辅助信息！

https://gz-metal-cell.github.io/2024/05/02/Hexo-%E4%BE%A7%E8%BE%B9%E6%A0%8F/

**Comment by promefire** [comment]
太强了😭
